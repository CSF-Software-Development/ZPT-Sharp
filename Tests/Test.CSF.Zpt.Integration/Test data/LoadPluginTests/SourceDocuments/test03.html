<html>
<head>
<title>Load plugin test 3</title>
</head>
<body>
<div>
  <ul>
    <li tal:repeat="item here/Items">
      <div tal:define="docName item/DocumentName;
                       docFullName string:${docName}.html;
                       doc load:here/Documents/?docFullName"
           tal:replace="doc">
        This content should be replaced by the rendered document
      </div>
    </li>
  </ul>
</div>
</body>
</html>