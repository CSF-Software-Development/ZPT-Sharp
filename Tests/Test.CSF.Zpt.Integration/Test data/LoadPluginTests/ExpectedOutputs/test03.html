<html>
<head>
<title>Load plugin test 3</title>
</head>
<body>
<div>
  <ul>
    <li>
      &lt;html&gt;
&lt;head&gt;
&lt;title&gt;Shared doc 1&lt;/title&gt;
&lt;/head&gt;
&lt;body&gt;
&lt;p&gt;
  Hello Joseph Bloggs!
  &lt;br&gt;Your item name is 'One'.
&lt;/p&gt;
&lt;/body&gt;
&lt;/html&gt;
    </li>
    <li>
      &lt;html&gt;
&lt;head&gt;
&lt;title&gt;Shared doc 2&lt;/title&gt;
&lt;/head&gt;
&lt;body&gt;
&lt;div&gt;
    &lt;p&gt;Content from the macro, before the slot-filler&lt;/p&gt;
    &lt;p&gt;
    Here's some slot-filling content
    &lt;br&gt;Your item name is 'Two'.
  &lt;/p&gt;
    &lt;p&gt;Content from the macro, after the slot-filler&lt;/p&gt;
  &lt;/div&gt;
&lt;/body&gt;
&lt;/html&gt;
    </li>
    <li>
      &lt;html&gt;
&lt;head&gt;
&lt;title&gt;Shared doc 3&lt;/title&gt;
&lt;/head&gt;
&lt;body&gt;
&lt;div&gt;
    &lt;p&gt;This is content from a different macro&lt;/p&gt;
    &lt;p&gt;
    Here's some slot-filling content
    &lt;br&gt;Your item name is 'Three'.
  &lt;/p&gt;
    &lt;p&gt;This is content from a different macro&lt;/p&gt;
  &lt;/div&gt;
&lt;/body&gt;
&lt;/html&gt;
    </li>
    
  </ul>
</div>
</body>
</html>